pkgs/applications/networking/mumble/default.nix

```diff
@@ -1,62 +1,39 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, qt5
+{ lib, stdenv, fetchFromGitHub, pkg-config, qt5, cmake
 , avahi, boost, libopus, libsndfile, protobuf, speex, libcap
 , alsa-lib, python3
 , rnnoise
 , jackSupport ? false, libjack2
 , speechdSupport ? false, speechd
 , pulseSupport ? false, libpulseaudio
-, iceSupport ? false, zeroc-ice
+, pipewireSupport ? false, pipewire
```

Comment:
Can we enable this by default? Many people are running native Pipewire already.

Reply:
Good point. Enabled Pulseaudio and Pipewire by default.

Reply:
Disabled them again since we enable Pipewire and Pulseaudio in all-packages.nix if `stdenv.isLinux` is true.

`meta.platforms` is set to `platforms.linux`, so they would be enabled in any case. I assume this was done for cross-platform support. So do we want to enable them by default?

Reply:
Looking at the git log, `platforms.linux` is used since at least 2015 and it seems nobody cared about cross-platform support. So I set these options to true and removed the settings from all-packages.nix.